README.md

```diff
@@ -71,3 +62,17 @@ If you find an issue with `odo`, please [file it here](https://github.com/redhat
 All our calls are open to public. You are welcome to join any of our calls.
```

Comment:
Taking inspiration from [Skaffold](https://github.com/GoogleContainerTools/skaffold/blob/main/README.md#community), I think it might be nice to add more welcoming message to people. Something like - 
>This is an open forum for anyone to show up and bring ideas, concerns, or just in general come hang out with the team! This is also a great time to get direct feedback on contributions, or give us feedback on ways you think we can improve the project. Come show us how you're using Skaffold!